monitor.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-mem=$(free -m | awk 'NR==2{print $3*100/$2}' | awk -F. '{print $1}')
+mem=$(free -m | awk 'NR==2{printf("%d\n", $3*100/$2)}')
```

Comment:
As in bash and other languages, `awk` supports `printf()`. In this case, `%d` is telling `printf()` to format the associated field (`$3*100/$2`) as an integer, not a floating point number.